packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFirstInvoice/index.mjml

```diff
@@ -0,0 +1,60 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-section css-class="mb-6">
+  <mj-column>
+    <mj-image src="<%- icon %>" alt="<%- productName %> icon" title="<%- productName %>" css-class="product-icon"></mj-image>
```

Comment:
Just a BTW, we have a follow up issue for localizing alt text, we aren't doing it at all in the old templates. This is good for now though👍